datasets/urdu_sentiment_corpus/README.md

```diff
@@ -0,0 +1,138 @@
+---
+annotations_creators:
+  - expert-generated
+language_creators:
+  - crowdsourced
+languages:
+  - ur
+licenses: []
+multilinguality:
+  - monolingual
+size_categories:
+  - 1K<n<10K
+source_datasets:
+  - original
+task_categories:
+  - text-classification
+task_ids:
+  - sentiment-classification
+---
+
+# Dataset Card for Urdu Sentiment Corpus (USC)
+
+## Table of Contents
+
+- [Dataset Description](#dataset-description)
+  - [Dataset Summary](#dataset-summary)
+  - [Supported Tasks](#supported-tasks-and-leaderboards)
+  - [Languages](#languages)
+- [Dataset Structure](#dataset-structure)
+  - [Data Instances](#data-instances)
+  - [Data Fields](#data-instances)
+  - [Data Splits](#data-instances)
+- [Dataset Creation](#dataset-creation)
+  - [Curation Rationale](#curation-rationale)
+  - [Source Data](#source-data)
+  - [Annotations](#annotations)
+  - [Personal and Sensitive Information](#personal-and-sensitive-information)
+- [Considerations for Using the Data](#considerations-for-using-the-data)
+  - [Social Impact of Dataset](#social-impact-of-dataset)
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+
+## Dataset Description
+
+- **Homepage:**
+- **Repository:** https://github.com/MuhammadYaseenKhan/Urdu-Sentiment-Corpus
+- **Paper:** https://ieeexplore.ieee.org/abstract/document/9080043
+- **Leaderboard:**
+- **Point of Contact:** <a href="https://github.com/MuhammadYaseenKhan">Muhammad Yaseen Khan</a>
+
+### Dataset Summary
+
+[More Information Needed]
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+[More Information Needed]
+
+## Dataset Structure
+
+### Data Instances
+
+[More Information Needed]
+
+### Data Fields
+
+- sentences: The Urdu tweet
+- sentiment: The sentiment that was exhibited in the tweet.
```

Comment:
can you detail a bit the different classes ?

Reply:
yeah sure